cmd/client/lib/testdata/drand_group.toml

```diff
@@ -0,0 +1,26 @@
+Threshold = 2
```

Comment:
could maybe use the existing `deploy/latest/group.toml` that's saved in the repo?